reservations/tasks.py

```diff
@@ -21,6 +22,7 @@
 def _prune_reservations() -> None:
     prune_inactive_reservations(PRUNE_OLDER_THAN_MINUTES)
     prune_reservation_with_inactive_payments(PRUNE_WITH_ORDERS_OLDER_THAN_MINUTES)
+    update_expired_orders(PRUNE_WITH_ORDERS_OLDER_THAN_MINUTES)
```

Comment:
I think this could be in its own task. What do you think? 🤔

Reply:
Moved! 👍